src/_nebari/stages/infrastructure/template/azure/modules/kubernetes/outputs.tf

```diff
@@ -17,3 +17,13 @@ output "kubeconfig" {
   sensitive   = true
   value       = azurerm_kubernetes_cluster.main.kube_config_raw
 }
+
+output "cluster_oidc_issuer_url" {
+  description = "The OpenID Connect issuer URL that is associated with the AKS cluster"
+  value       = azurerm_kubernetes_cluster.main.oidc_issuer_url
+}
```

Comment:
When `oidc_issuer_enabled` is False, does this output return null, or does terraform raises an error?

Reply:
It returns null